nodejs/examples/vpc/index.ts

```diff
@@ -53,4 +53,14 @@ const vpcWithLocations = new awsx.ec2.Vpc("custom6", {
         { type: "isolated", name: "db", location: "10.0.2.0/24" },
         { type: "isolated", name: "redis", location: "10.0.3.0/24" },
     ],
-}, providerOpts);
\ No newline at end of file
+}, providerOpts);
+
+const vpcWithSpecificZones = new awsx.ec2.Vpc("custom7", {
+    requestedAvailabilityZones: ["us-east-1a","us-east-1b","us-east-1c"],
+    subnets: [
+        { type: "public"},
+        { type: "private"},
+        { type: "isolated", name: "db"},
+        { type: "isolated", name: "redis"},
```

Comment:
space before `}`

Reply:
Done, did I miss that you have formatter rules of some sort? I had to disable my own on save since it would change everything, but a lot of these could be resolved if you introduced one that vscode would respect.